### PR TITLE
Make pre-fork and post fork spec tests pass 

### PR DESCRIPTION
**Pull request by AKorpusenko**

### CODE REVIEW

File: operator/validator/controller.go [hunk 1]
```diff
@@ -75,7 +75,6 @@ type ControllerOptions struct {
 	MetadataUpdateInterval     time.Duration `yaml:"MetadataUpdateInterval" env:"METADATA_UPDATE_INTERVAL" env-default:"12m" env-description:"Interval for updating metadata"`
 	HistorySyncBatchSize       int           `yaml:"HistorySyncBatchSize" env:"HISTORY_SYNC_BATCH_SIZE" env-default:"25" env-description:"Maximum number of messages to sync in a single batch"`
 	MinPeers                   int           `yaml:"MinimumPeers" env:"MINIMUM_PEERS" env-default:"2" env-description:"The required minimum peers for sync"`
-	NetworkConfig              networkconfig.NetworkConfig
```
[the remark this answers is not in the document]
Comment (AKorpusenko) [reply operator/validator/controller.go:78]: `NetworkConfig              networkconfig.NetworkConfig`
There were 2 identical lines. I removed one of them

File: ekm/genesis_adapter.go [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package ekm
 
 import (
+	genesisphase0 "github.com/AKorpusenko/genesis-go-eth2-client/spec/phase0"
```
Comment (y0sher) [inline ekm/genesis_adapter.go:4]: did `go-eth2-client/spec/phase0` change?
also if we're using forked code, lets at least use "replace" directive, and preferably lets put it in a repo under ssv.

File: ekm/genesis_adapter.go
Comment (y0sher) [inline ekm/genesis_adapter.go:1]: I don't really understand why we need this file

File: logging/fields/fields.go [hunk 3]
```diff
@@ -220,15 +222,25 @@ func OperatorIDStr(operatorId string) zap.Field {
...
 	return zap.Uint64(FieldHeight, uint64(height))
 }
+func GenesisHeight(height genesisspecqbft.Height) zap.Field {
+	return zap.Uint64(FieldHeight, uint64(height))
+}
 
 func Round(round specqbft.Round) zap.Field {
 	return zap.Uint64(FieldRound, uint64(round))
 }
+func GenesisRound(round genesisspecqbft.Round) zap.Field {
```
Comment (y0sher) [inline logging/fields/fields.go:232]: probably Round will just work as the types are compatible and can be cast easily, by minimum move these to separate file.

File: ekm/genesis_adapter.go (hunk 2 above)
Comment (GalRogozinski) [reply ekm/genesis_adapter.go:4]: if this is the reason for the file it should be removed once pec version is upgraded

**Comment by AKorpusenko** [comment]
Replaced with https://github.com/ssvlabs/ssv/pull/1503

**Pull request closed without merging**